src/database/SsiCredentialIssuer.Migrations/Seeder/Data/verified_credential_external_type_detail_versions.json

```diff
@@ -5,78 +5,86 @@
     "version": "3.0",
     "template": "https://catena-x.net/fileadmin/user_upload/04_Einfuehren_und_umsetzen/Governance_Framework/231016_Catena-X_Use_Case_Framework_Traceability.pdf",
     "valid_from": "2024-01-01 00:00:00.000000 +00:00",
-    "expiry": "2025-06-30 00:00:00.000000 +00:00"
+    "expiry": "2024-10-16 00:00:00.000000 +00:00"
   },
   {
     "id": "1268a76a-ca19-4dd8-b932-01f24071d565",
     "verified_credential_external_type_id": 5,
     "version": "1.0",
     "template": "https://catena-x.net/fileadmin/user_upload/04_Einfuehren_und_umsetzen/Governance_Framework/231016_Catena-X_Use_Case_Framework_CircularEconomy.pdf",
     "valid_from": "2024-01-01 00:00:00.000000 +00:00",
-    "expiry": "2024-10-24 00:00:00.000000 +00:00"
+    "expiry": "2024-10-16 00:00:00.000000 +00:00"
   },
   {
     "id": "37aa6259-b452-4d50-b09e-827929dcfa15",
     "verified_credential_external_type_id": 6,
     "version": "1.0",
     "template": "https://catena-x.net/fileadmin/user_upload/04_Einfuehren_und_umsetzen/Governance_Framework/231016_Catena-X_Use_Case_Framework_PCF.pdf",
     "valid_from": "2024-03-27 00:00:00.000000 +00:00",
-    "expiry": "2024-12-31 00:00:00.000000 +00:00"
+    "expiry": "2024-10-16 00:00:00.000000 +00:00"
   },
   {
     "id": "27d12475-c970-4979-892b-8f88e819018f",
     "verified_credential_external_type_id": 8,
     "version": "1.0",
     "template": "https://catena-x.net/fileadmin/user_upload/04_Einfuehren_und_umsetzen/Governance_Framework/231016_Catena-X_Use_Case_Framework_DemanAndCapacity.pdf",
     "valid_from": "2024-03-27 00:00:00.000000 +00:00",
-    "expiry": "2024-10-24 00:00:00.000000 +00:00"
+    "expiry": "2024-10-16 00:00:00.000000 +00:00"
   },
   {
     "id": "a7585e82-4789-47ce-9184-5788086b1943",
     "verified_credential_external_type_id": 9,
     "version": "1.0",
     "template": "https://catena-x.net/fileadmin/user_upload/04_Einfuehren_und_umsetzen/Governance_Framework/231016_Catena-X_Use_Case_Framework_Puris.pdf",
     "valid_from": "2024-03-27 00:00:00.000000 +00:00",
-    "expiry": "2024-10-24 00:00:00.000000 +00:00"
+    "expiry": "2024-10-16 00:00:00.000000 +00:00"
   },
   {
     "id": "0077addf-f50d-4f5e-bc41-26c45d407104",
     "verified_credential_external_type_id": 10,
     "version": "1.0",
     "template": "https://catena-x.net/fileadmin/user_upload/04_Einfuehren_und_umsetzen/Governance_Framework/231016_Catena-X_Use_Case_Framework_BPDM.pdf",
     "valid_from": "2024-03-27 00:00:00.000000 +00:00",
-    "expiry": "2024-10-24 00:00:00.000000 +00:00"
+    "expiry": "2024-10-16 00:00:00.000000 +00:00"
+  },
+  {
+    "id": "090efafd-9667-404f-85cc-d7d072b5ad46",
+    "verified_credential_external_type_id": 12,
+    "version": "1.0",
+    "template": "https://catena-x.net/fileadmin/user_upload/04_Einfuehren_und_umsetzen/How_To_Conclude_Data_Exchange_Contracts.pdf",
+    "valid_from": "2024-10-16 00:00:00.000000 +00:00",
+    "expiry": "2025-10-16 00:00:00.000000 +00:00"
```

Comment:
@ntruchsess gibt es die Option das Datum auf unendlich zu setzen? E.g.: "1.1.2099"

Reply:
done, set to 9999-12-31 23:59:59.000000 +00:00